.circleci/config.yml

```diff
@@ -16,8 +16,52 @@ jobs:
       - run:
           name: "Sending coverage reports to CodeCov"
           command: "bash <(curl -s https://codecov.io/bash)"
+
+  build-docs:
+      docker:
+      - image: circleci/python:3.7.3
+      steps:
+      - checkout
+      - run:
+          name: "Installing Dependencies"
+          command: "sudo pip3 install -r requirements.txt"
+      - run:
+          name: "Setting environment variable"
+          command: "export PYTHONWARNINGS='error::UserWarning'"
+      - run:
+          name: "Building documenation using pdoc3"
+          command: "pdoc --html --skip-errors --output-dir docs ."
+      - persist_to_workspace:
+          root: ~/project/docs
+          paths: project
+
+  docs-deploy:
+      docker:
+      - image: node:8.10.0
+      steps:
+      - checkout
+      - add_ssh_keys:
+            fingerprints:
+                - "7f:f7:8d:79:e6:17:7a:4c:e3:ec:b7:05:55:57:fe:26"
```

Comment:
You should not add your fingerprints like this to the repo. Ideally all secrets should be stored as environment variables in the CICD platform (CircleCI here)

Reply:
Quoting [this article](https://circleci.com/blog/deploying-documentation-to-github-pages-with-continuous-integration/) from the CircleCI website:

"fingerprint for our key, which is a unique identifier that’s safe to expose publicly (unlike the private key itself..."

Let me know if we still need to add an environment variable.

Reply:
By whose finger print is this ? And why should it be added to this repo....that's the question.

Reply:
This is actually "CircelCI's fingerprint" but was added in my organisation. This fingerprint is generated when the private key is handed over to CircleCI. This needs to be done for Intuit organisation. [Steps here.](https://circleci.com/docs/2.0/gh-bb-integration/#deployment-keys-and-user-keys)

Also, for some reason build failed when I tried to replace this with env variable on CircleCI. When specified back directly, the build passed. We can update this later when Intuit org has created a fingerprint.